### PR TITLE
Create zh_cn.json

### PR DESCRIPTION
**Pull request by Neubulae**
I'll post this on Fabric Translation Discord for pending reviews to see which one's better.
(cmon qseft why every time hmpf

### CODE REVIEW

**Comment by Neubulae** [comment]
hangon what

**Comment by Neubulae** [comment]
I thought it retained the look and feel, so sad man

**Review by LeoCTH — APPROVED** [review]
草 IGN10/10

File: src/main/resources/assets/gofish/lang/zh_cn.json [hunk 1]
```diff
@@ -1,115 +1,114 @@
...
-    "item.gofish.obsidian_halibut":"Obsidian Halibut",
-    "item.gofish.oakfish":"Oakfish",
-    "item.gofish.nebula_swordfish":"Nebula Swordfish",
-    "item.gofish.matrix_rod":"Matrix Rod",
-    "item.gofish.matrix_fish":"Matrix Fish",
-    "item.gofish.magma_cod":"Magma Cod",
-    "item.gofish.lunarfish":"Lunarfish",
-    "item.gofish.lilyfish":"Lilyfish",
-    "item.gofish.icicle_fish":"Icicle Fish",
-    "item.gofish.grilled_blackstone_trout.tooltip_2":"is savory and filling.",
```
Comment (LeoCTH) [inline src/main/resources/assets/gofish/lang/zh_cn.json]: leave it empty
` "item.gofish.grilled_blackstone_trout.tooltip_2":"",`

**Review by Hambaka — APPROVED** [review]

**Comment by Draylar** [comment]
Awesome, thanks for all your effort put into this! Released in a second here.

**Pull request merged**